atests/Screenshot.robot

```diff
@@ -74,6 +74,17 @@ No Screenshots Should Created For No Library Keywords
     File Should Not Exist    ${OUTPUT DIR}/${SCREENSHOT_FOLDER}/${FILENAME}
     Set Screenshot Directory
 
+Take Screenshot As Base64
+    Set Screenshot Log Mode    Base64
+    Take Screenshot
+
+Take Screenshot Of Window

```

Comment:
I have not really grasped well how you structure your test cases. These 2 fail, since I'm not launching the application properly yet.